morgan/tests/mod_test.ts

```diff
@@ -0,0 +1,24 @@
+import { Rhum } from "../../test_deps.ts";
+import { Morgan } from "../mod.ts";
+
+Rhum.testPlan("mod.ts", () => {
+  Rhum.testSuite("Morgan", () => {
```

Comment:
Can help but feel we should test it fully works eg the output, you probably thought of that and idk how it'd be done in a clean way (maybe passing in empty drash request, calling the logger and asserting the stdout?)

Reply:
doneski. testing the output for all log methods now